frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/behaviors/transfer-vehicles/simulated-region-overview-behavior-transfer-vehicles.component.ts

```diff
@@ -119,6 +132,17 @@ export class SimulatedRegionOverviewBehaviorTransferVehiclesComponent
     }
 
     ngOnInit(): void {
+        if (this.transferToStart) {
+            this.transferCollapsed = false;
+            this.vehicleToSend = this.transferToStart.vehicleToTransfer;
+            this.selectedDestination = this.transferToStart.transferDestination;
+            this.clickedPatients =
```

Comment:
These three variable names should be similar, to make it clear, that they belong together.
How about `selectedVehicle`, `selectedDestination` and `selectedPatients`?

Reply:
Done